extension-config/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
+  "manifest_version": 2,
+  "name": "tota11y",
+  "description": "An accessibility visualization toolkit",
+  "version": "1.0",
```

Comment:
Should we match the version to our npm package version?

Reply:
If so, should we add some scripting to ensure that versions stay aligned?

Reply:
I would bump the version, currently 0.1.6 on npm and 0.2.0 in the package.json to 0.3.0 since the changes are minor in the sense there are no breaking changes, just additional functionality via the browser extension. I ran `npm version minor` to do that. I'd suggest just running `npm version x` based on what you want to do in terms of [npm versioning](https://docs.npmjs.com/cli/version) before publishing. I'm happy to add some more scripting, but I'm not sure  much more is needed between `npm version` and the [pre-publish-checks.js](https://github.com/Khan/tota11y/blob/master/utils/pre-publish-checks.js). Thoughts?